aii-opennebula/src/main/resources/tests/profiles/vm.pan

```diff
@@ -97,7 +105,8 @@ bind "/system/opennebula" = opennebula_vmtemplate;
 prefix "/system/opennebula";
 "vnet" = nlist(
     "eth0", "br100",
-    "eth1", "br101");
+    "eth1", "br101",
+    "eth2", "br101");
 
```

Comment:
@stdweird ok I see, if the hardware mac is always present in quattor  what about include ignoremac as a boolean type to be used by AII configure hook?:
```pan
"ignoremac" ? boolean = false # use quattor hw mac address instead OpenNebula mac prefix
```
but this will affect all the VM ARs, what do you think?